routes.py

```diff
@@ -1,11 +1,12 @@
 from flet_route import path
-from views import indexView, aboutAppView, aboutAppCity, museumInfoView, settings
+from views import indexView, aboutAppView, aboutAppCity, museumInfoView, settings, eventViews
 
 app_routes = [
     path(url="/", clear=True, view=indexView),
     path(url="/mus/:id", clear=False, view=museumInfoView),
     path(url='/about_app/', clear=False, view=aboutAppView),
     path(url='/about_city/', clear=False, view=aboutAppCity),
     path(url='/settings/', clear=False, view=settings),
+    path(url='/events/', clear=False, view=eventViews),
```

Comment:
А event у тебя разве не динамический будет? Как например ```/mus/:id/```

Reply:
нет так как там будет список так же как и на главной страничке

Reply:
А на каждое событие можно будет перейти?

Reply:
да. Вот там как раз и будет динамическая. Всё как с музеями.

Reply:
В таком случае нужно будет как саму страницу с событиями сделать, так и динамическую страничку для каждого события.
```suggestion
    path(url='/events/', clear=False, view=eventViews),
    path(url='/events/:id', clear=False, view=eventInfoView)```

Reply:
а зачем её делать денамической? Если мне надо будет другую стр делать денамической.